lib/apartment/active_record/postgresql_adapter.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# rubocop:disable Style/ClassAndModuleChildren
+
+# NOTE: This patch is meant to remove any schema_prefix appart from the ones for
+# excluded models. The schema_prefix would be resolved by apartment's setting
+# of search path
+module Apartment::PostgreSqlAdapterPatch
+  def default_sequence_name(table, _column)
+    res = super
+    schema_prefix = "#{Apartment::Tenant.current}."
+
+    if res&.starts_with?(schema_prefix)
+      if Apartment.excluded_models.none? { |m| m.constantize.table_name == table }
```

Comment:
I find it hard to parse `if/else` with negatives (`none?`). I would use `any?` and swap the bodies